app/decorators/event_decorator.rb

```diff
@@ -28,6 +28,6 @@ def description
   end
 
   def details_link
-    link_to t('events.banner.more_details'), object, class: 'btn btn-success'
+    link_to t('events.banner.more_details'), object, class: 'btn-more-details'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.